src/Appwrite/Platform/Workers/Webhooks.php

```diff
@@ -113,7 +121,21 @@ private function execute(array $events, string $payload, Document $webhook, Docu
         }
 
         if (false === \curl_exec($ch)) {
-            $this->errors[] = \curl_error($ch) . ' in events ' . implode(', ', $events) . ' for webhook ' . $webhook->getAttribute('name');
+            $dbForConsole->increaseDocumentAttribute('webhooks', $webhook->getId(), 'attempts', 1);
```

Comment:
If error increases attempts by 1, successful request should reset attempts back to 0.

If I have my webhook and it fails 10 times in 1 year after 1 million requests, it should not get blocked.